demos/benchmark/lv_demo_benchmark.c

```diff
@@ -663,6 +665,39 @@ void lv_demo_benchmark(void)
     scene_next_task_cb(NULL);
 }
 
+
+void lv_demo_benchmark_run_scene(int_fast16_t scene_no)
+{
+    benchmark_init();
+
+    if(((scene_no >> 1) >= dimof(scenes))) {
+        /* invalid scene number */
+        return ;
+    }
+
+    opa_mode = scene_no & 0x01;
+    scene_act = scene_no >> 1;
+
+    if(scenes[scene_act].create_cb) {
+        lv_label_set_text_fmt(title, "%"LV_PRId32"/%d: %s%s", scene_act * 2 + (opa_mode ? 1 : 0), (int)(dimof(scenes) * 2) - 2,
+                              scenes[scene_act].name, opa_mode ? " + opa" : "");
+        lv_label_set_text(subtitle, "");
+
+        rnd_reset();
+        scenes[scene_act].create_cb();
+
+        lv_timer_t * t = lv_timer_create(report_cb, SCENE_TIME, NULL);
+        lv_timer_set_repeat_count(t, 1);
+    }
+}
+
+
+void lv_demo_benchmark_register_finished_handler(finished_cb_t * finished_cb)
```

Comment:
I think this name would be more similar to other API names:
```suggestion
void lv_demo_benchmark_set_finished_cb(finished_cb_t * finished_cb);
```

Reply:
OK, I have no problem with your suggestion. Let's make this change.